README.md

```diff
@@ -63,9 +77,11 @@ sudo docker push sydneyinformaticshub/rnaseq-rstudio:4.1.0
 
 ## Run with Singularity 
 
-If you are on a machine with no Docker (like a HPC environment), you may wish to use [Singularity](https://docs.sylabs.io/guides/3.7/admin-guide/installation.html) as an alternative.
+If you are on a machine with no Docker (like a HPC environment or a Pawsey Nimbus instance), you may wish to use [Singularity](https://docs.sylabs.io/guides/3.7/admin-guide/installation.html) as an alternative.
+
+The following steps can assist in running the container on a Pawsey NImbus instance.
```

Comment:
Remove this line. Keep formatting and style consistent with the rest of your README. Replace with: 

```
### On Pawsey Nimbus cloud
```

Reply:
DONE